### ISSUE TITLE
Every time the server is rebooted the integration with AzureAD stops working. To make it work again, it is needed to access AF with a local admin account, go to "Settings => MS Entra ID" and click on the UPDATE button.

### ISSUE THREAD

**Issue by dariopai** [ansibleguy76/ansibleforms#229]
**Describe the bug**
Every time the server is rebooted the integration with AzureAD stops working. To make it work again, it is needed to access AF with a local admin account, go to "Settings => MS Entra ID" and click on the UPDATE button.

**To Reproduce**
1. Reboot the server
2. Login to AF using AzureAD credentials
![error_afv504_and_msentraid_browsing005_error_after_server_reboot](https://github.com/user-attachments/assets/7d853f7e-d6e8-45a7-ac7a-47fd75ec8212)
5. To fix it => Sertings => Auth MS Entra ID => click on UPDATE button
![image](https://github.com/user-attachments/assets/89538bb9-ee7e-4b55-bba5-9a89ee3203ec)

**Expected behavior**
AzureAD Auth should continue working after a reboot.

**Version**
ansibleforms v4.0.19 and also with v5.0.4

**Deployment**
Deployed ansibleforms with :
- kubernetes 1.27/1.28/1.29

**Comment by ansibleguy76** [comment]
ah... i believe i know the issue.  It's probably your app starting faster than your database.  Check you startup logs.  It's quite possible that the application tries to load the azuread authentication settings, and the database is not started yet.  That's somewhat strange because version 5.0.3 tried to fix that by wait to initialize until the database is reachable.  

  // this is at startup, don't start the app until mysql is ready
  // rewrite with await
  logger.info("Waiting for mysql to start")
  async function sleep(millis) {
    return new Promise(resolve => setTimeout(resolve, millis));
  }
  var MYSQL_IS_READY = false
  while(!MYSQL_IS_READY){
    try{
      await mysql.do("SELECT 1")
      MYSQL_IS_READY = true
    }catch(e){
      logger.log("Mysql not ready yet")
      await sleep(5000)
    }
  }

**Comment by ansibleguy76** [comment]
can you check if you see this in the logs, the moment the app starts ?

```
2024-09-24 12:50:45:5045 info: Waiting for mysql to start
2024-09-24 12:50:45:5045 notice: Serving static files from /app/dist/views
2024-09-24 12:50:45:5045 notice: Exposing app under /
2024-09-24 12:50:45:5045 notice: Running https !
2024-09-24 12:50:45:5045 notice: App running on port 443!
2024-09-24 12:50:45:5045 info: Mysql is ready
2024-09-24 12:50:45:5045 info: Schema is up to date
2024-09-24 12:50:45:5045 info: Moving older form backups to new backup folder
2024-09-24 12:50:45:5045 notice: Abandoning jobs
2024-09-24 12:50:45:5045 info: Initializing hourly abandoned jobs timer
2024-09-24 12:50:45:5045 info: Pulling repositories
2024-09-24 12:50:45:5045 info: Initializing repository cron schedules
2024-09-24 12:50:45:5045 warning: Failed to generate ssh keys : already exists
2024-09-24 12:50:46:5046 info: Azure AD strategy initialized
```

**Comment by ansibleguy76** [comment]
this would make sense, because hitting the update button, loads the azure provider.  The one during init somehow fails, my best guess would be that the database is not ready yet.  I'll wait for your feedback.  If you can share the logs ?  and if possible have debug enabled + ENABLE_DB_QUERY_LOGGING = true => this will log all the mysql queries.

**Comment by dariopai** [comment]
I have just rebooted my PROD K8s server and I can confirmed that the lines "Initializing Azure AD strategy...Adding the strategy azure ad" are never shown in the console log. 
 
Only once I click the button UPDATE at SETTINGS =>MS ENTRA ID these lines appear at the console. So, in my escenario, it seems that the following code is no executed:

![image](https://github.com/user-attachments/assets/2e769b58-6063-4747-b744-55328dfafa06)

Lines when clickin UPDATE button at MS ENTRA ID section:
![image](https://github.com/user-attachments/assets/51a708f0-c946-4c00-ade8-adcebc8a2efd)

**Comment by dariopai** [comment]
Since rebooted the sequence of code execution when login with AzureAD seems to follow this steps:

![image](https://github.com/user-attachments/assets/f4720300-b53d-4235-bde8-23f5c0d17ab6)

**Comment by ansibleguy76** [comment]
correct. i believe so too. but that would indicate your database is not ready.  maybe i should do a some more tests like checking the azuread table.

**Comment by dariopai** [comment]
my guess is that pod "coredns" is not solving properly the ip address for pod "mysql" while K8s is initializing ... so this situation blocks the code and the "while not connecting to mysql" never executes:

![image](https://github.com/user-attachments/assets/c9be759b-c69c-443c-a2dd-c81ad87ccf4f)

**Comment by ansibleguy76** [comment]
problem found and fixed in new beta...

**Comment by ansibleguy76** [comment]
fixed in new release 5.0.7
